tasks/self-introduction/self-introduction.md

```diff
@@ -1,59 +1,59 @@
+The presentation must be **in English**. In addition to recording a video, you need to make a text transcript of the speech (also **in English**). Place a link to the video and transcript in [private school repository](https://docs.rs.school/#/private-repository?id=Как-работать-с-приватным-репозиторием) and [deploy using gh-pages](Как-сделать-деплой-задания-из-приватного-репозитория-школы).
 
-## Порядок работы
+## Task implementation
 
-1. Запишите видео с вашим выступлением на английском языке. Желательно, чтобы ваше лицо было в кадре (для записи можно использовать веб-камеру, камеру телефона, видеокамеру, камеру ноутбука и т.д).
-2. Загрузите видео в ваш личный [YouTube](https://www.youtube.com/)-аккаунт. Можно сделать видеоролик доступным только по ссылке — для этого в настройках видео установите [Параметры доступа > Доступ по ссылке (Visibility > Unlisted)](https://support.google.com/youtube/answer/157177?co=GENIE.Platform%3DDesktop&hl=en).
-3. В вашем **приватном** репозитории (по ссылке вида `https://github.com/rolling-scopes-school/your-github-account-JSFE2022Q3`) от ветки `main` создайте ветку `self-introduction`.
-4. В созданной ветке `self-introduction` создайте папку `self-introduction`, в ней создайте файл `index.md`.
-5. Добавьте ссылку на ваше YouTube-видео в файл `index.md`
-6. Под ссылкой добавьте текстовую расшифровку вашего выступления. Текст должен повторять сказанное в вашем выступлении, но может быть не дословным — можно отредактировать текст, убрав из него лишние междометия, слова-паразиты и т.п.
-7. Для деплоя используйте gh-pages [Как сделать деплой задания из приватного репозитория школы](https://docs.rs.school/#/private-repository?id=Как-сделать-деплой-задания-из-приватного-репозитория-школы)
-8. Сделайте Pull Request из ветки `self-introduction` в ветку `main`. Pull Request должен называться так же, как и задание — **Self-Introduction**, и соответствовать [требованиям к Pull Requests](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr). В качестве скриншота результата можно прикрепить скриншот видео, а в качестве ссылки на деплой — ссылку вида `https://rolling-scopes-school.github.io/your-github-account-JSFE2022Q3/self-introduction/`. **Мержить Pull Request не нужно**.
+1. Record your presentation in English. It is desirable that your face is in the frame (for recording, you can use a webcam, phone camera, video camera, laptop camera, etc.).
+2. Upload the video to your personal [YouTube](https://www.youtube.com/) account. You can make a video available only via a link - to do this, in the video settings, set [Параметры доступа > Доступ по ссылке (Visibility > Unlisted)](https://support.google.com/youtube/answer/157177?co=GENIE.Platform%3DDesktop&hl=en).
```

Comment:
It's allowed to make a video available only via a link.
[Video details -> Visibility -> Unlisted](https://support.google.com/youtube/answer/157177?co=GENIE.Platform%3DDesktop&hl=en)
Let's add this link https://www.youtube.com/watch?v=JOr7JluzEOM&ab_channel=MN as instructions as well